### DISCUSSION TITLE
🤯 Guidelines for Issue Submission and FAQ

### DISCUSSION THREAD

**Discussion by canisminor1990** [lobehub/sd-webui-lobe-theme#469] [Announcements]
> [!IMPORTANT]
> 
> **Before submitting, please check:**
> 
> 1. Ensure that your plugin branch is `main` and it is the **latest commit** (There have been multiple instances of updates lagging due to switching to the `dev` or other branches)
> 2. Check if the SD WebUI is the **latest version**
> 3. Try **disabling other plugins** to rule out conflicts causing the issue; if so, please provide the `name of the conflicting plugin`
> 4. There are no current plans to support third-party modified versions such as `SD Next`, etc.
> 5. If extra-network loading infinite, try adding `--no-gradio-queue` to the COMMANDLINE_ARGS

> [!TIP]
>
> **The versions currently released have all undergone standard environment testing during the submission period:**
> - Browser environment: Latest version of Chrome
> - SD WebUI version: The latest version from the official website at the time of the plugin's release
> - Plugins: ControlNet (This is for minimum environment testing)

> [!NOTE]
> 
> **Follow the rules:**
> 
> - Before submitting an issue, please search existing `issues` to ensure that the problem has not already been raised
> - Ensure that the problem description is clear, accurate, and includes relevant environmental information and error logs
> - Please adhere to a friendly and polite code of conduct; do not post malicious, aggressive, or irrelevant comments
> - If an issue has been resolved, please close the corresponding `issue`
> - If you are capable of resolving the issue, please submit the corresponding `pull request`

---

> [!IMPORTANT]
> 
> **提交前请检查：**
> 
> 1. 你的插件分支是否为 `main` 同时为 **最新一次提交** (发现有多例因为切换到 `dev` 或其他分支而造成的更新滞后的问题)
> 2. 请检查 SD WebUI 是否为 **最新版本**
> 3. 通过 **禁用其他插件** 来排除是否由于插件冲突所导致的问题，如是则请提供 `产生冲突插件的名称`
> 4. 暂无计划支持第三方魔改版本，如 `SD Next` 等
> 5. 如果遇到模型无限加载的情况, 尝试通过 `--no-gradio-queue` 禁用 Gradio 队列

> [!TIP]
>
> **目前发布的版本，均在提交时段完成标准环境测试：**
> - 浏览器环境：最新版 Chrome
> - SD WebUI 版本：官网截止插件发布时的最新版本
> - 插件：ControlNet (此为最小环境检测)

> [!NOTE]
> 
> **遵守规则：**
> 
> - 提交问题前，请先搜索已有的 `issues` 确保问题尚未被提出
> - 请确保问题描述清晰、准确，并提供相关的环境信息和错误日志
> - 请遵守友好、礼貌的行为准则，不要发布恶意、攻击性或无关的评论
> - 如果问题已解决，请关闭对应的 `issue`
> - 如果你有能力解决问题，请提交相应的 `pull request`

**Comment by ahem95** [comment]
There is a bug that the quick settings are constantly loading
![1](https://github.com/lobehub/sd-webui-lobe-theme/assets/119165594/03d3bd8d-e4d6-4bef-9399-7c35ef90f2e7)
